packages/nimble-components/src/rich-text/editor/tests/rich-text-editor-matrix.stories.ts

```diff
@@ -63,7 +63,7 @@ const component = (
         ${() => footerHiddenName} ${() => errorStateName} ${() => placeholderName} ${() => disabledName} 
     </p>
     <${richTextEditorTag}
-        style="margin: 5px 0px; width: 500px;"
+        style="margin: 5px 0px; width: 500px; height: 100px"
```

Comment:
The changes to the component height in this PR are making it difficult to inspect the Chromatic diff and see what changed related to links. What do you think about splitting them into a separate PR so that the link changes are isolated? We could push that one through quickly first thing Tuesday morning (NIC time).

Reply:
Created a PR for setting the heights in the storybook. https://github.com/ni/nimble/pull/1513